### DISCUSSION TITLE
Welcome to rsuite-table Discussions!

### DISCUSSION THREAD

**Discussion by simonguo** [rsuite/rsuite-table#265] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shinenic** [comment]
Hi @simonguo,
this library looks pretty good and very smooth!
I would like to ask whether any plan to add API to enable fixed/frozen rows?
like [this](https://autodesk.github.io/react-base-table/examples/frozen-rows)

thanks!
